lib/manageiq/loggers/cloud_watch.rb

```diff
@@ -0,0 +1,29 @@
+require 'active_support/core_ext/string'
+
+module ManageIQ
+  module Loggers
+    module CloudWatch
+      NAMESPACE_FILE = "/var/run/secrets/kubernetes.io/serviceaccount/namespace".freeze
+
+      def self.new(*args)
+        access_key_id     = ENV["CW_AWS_ACCESS_KEY_ID"].presence
+        secret_access_key = ENV["CW_AWS_SECRET_ACCESS_KEY"].presence
+        log_group_name    = ENV["CLOUD_WATCH_LOG_GROUP"].presence
+        log_stream_name   = File.exist?(NAMESPACE_FILE) ? File.read(NAMESPACE_FILE) : nil
+
+        unless access_key_id && secret_access_key && log_group_name && log_stream_name
+          return ManageIQ::Loggers::Container.new 
+        end
+
+        require 'cloudwatchlogger'
+
+        creds = {:access_key_id => access_key_id, :secret_access_key => secret_access_key}
+        cloud_watch_logdev = CloudWatchLogger::Client.new(creds, log_group_name, log_stream_name)
+
+        ManageIQ::Loggers::Base.new(cloud_watch_logdev).tap do |logger|
```

Comment:
I think it might be cleaner to have a `class Cloudwatch < Base`, and just call `super` here.  Then have

```ruby
def initialize(*args)
  super
  self.formatter = ManageIQ::Loggers::Container::Formatter.new
end
```